Upstream-Status: Backport

Add missing includes required when building with modern toolchain, based on
patch from Debian bugzilla:
http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=505954

Should no longer be required once upgraded to 0.7.22 or later.

Index: apt-0.7.14/apt-pkg/acquire.cc
===================================================================
--- apt-0.7.14.orig/apt-pkg/acquire.cc	2008-05-28 14:22:13.000000000 +0100
+++ apt-0.7.14/apt-pkg/acquire.cc	2010-07-23 17:30:11.494883936 +0100
@@ -22,6 +22,7 @@
 
 #include <apti18n.h>
 
+#include <cstdio>
 #include <iostream>
 #include <sstream>
     
Index: apt-0.7.14/apt-pkg/contrib/sha256.h
===================================================================
--- apt-0.7.14.orig/apt-pkg/contrib/sha256.h	2008-05-28 14:22:14.000000000 +0100
+++ apt-0.7.14/apt-pkg/contrib/sha256.h	2010-07-23 17:30:11.494883936 +0100
@@ -14,6 +14,7 @@
 #ifndef APTPKG_SHA256_H
 #define APTPKG_SHA256_H
 
+#include <stdint.h>
 #include <string>
 #include <cstring>
 #include <algorithm>
Index: apt-0.7.14/apt-pkg/deb/dpkgpm.cc
===================================================================
--- apt-0.7.14.orig/apt-pkg/deb/dpkgpm.cc	2008-05-28 14:22:14.000000000 +0100
+++ apt-0.7.14/apt-pkg/deb/dpkgpm.cc	2010-07-23 17:30:36.960856870 +0100
@@ -20,6 +20,7 @@
 #include <stdlib.h>
 #include <fcntl.h>
 #include <sys/select.h>
+#include <sys/stat.h>
 #include <sys/types.h>
 #include <sys/wait.h>
 #include <signal.h>
